Presentation/CustomAdaptivePresentationDelegate.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  CustomAdaptivePresentationDelegate.swift
+//  Presentation
+//
+//  Created by Vasil Blanco-Nunev on 2019-07-19.
+//  Copyright © 2019 iZettle. All rights reserved.
+//
+
+import UIKit
+import Flow
+
+public final class CustomAdaptivePresentationDelegate: NSObject, UIAdaptivePresentationControllerDelegate {
```

Comment:
Do we want this to be available for < iOS 13.0?

Should we move the conformance to UIAdaptivePresentationControllerDelegate to the extension where we implement it?

Reply:
The protocol has few iOS 8 methods, added them too. Prefer to keep the iOS 13 only signals here as well, they won't signal before iOS 13 but no need of a check on the call side too.